inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -107,8 +110,13 @@ void LayerTestsCommon::QueryNetwork() {
     ASSERT_EQ(expected, actual);
 }
 
-InferenceEngine::Blob::Ptr LayerTestsCommon::GenerateInput(const InferenceEngine::InputInfo &info) const {
-    return FuncTestUtils::createAndFillBlob(info.getTensorDesc());
+InferenceEngine::Blob::Ptr LayerTestsCommon::GenerateInput(const InferenceEngine::InputInfo& info) const {
+    return FuncTestUtils::createAndFillBlob(targetStaticShape.empty() || targetStaticShape[0].empty() ?
```

Comment:
As for me, we should set `targetStaticShape` from input shapes if `dynamicShapes` is empty

Reply:
This code is added to support tests which do not support input shapes but use LayerTestsCommon.

Reply:
and what about - `targetStaticShape`? May be `targetStaticShapes`?

Reply:
It's like;
for (tss : targetStaticShapes) {
  targetStaticShape = tss;
}
It's for multiple target shapes and to keep codes untouched as possible, so static variable targetStaticShape is used.

Reply:
> > And, again, ask from @mandrono .. Is it works correct for nodes with more than 1 input??
> 
> I guess so, but as I had no such cases and just focus on implementing features, they were not tested separately.
I think that we should provide a example and check it in the future

Reply:
Sure